CMakeLists.txt

```
@@ -73,5 +73,8 @@ install(TARGETS cpptcl_static ARCHIVE DESTINATION lib)
 install(FILES ${HDRS} DESTINATION include/cpptcl)
 install(FILES ${HDRS_DETAILS} DESTINATION include/cpptcl/details)
 
+add_library(cpptcl_runtime SHARED ${cpptcl_SOURCE_DIR}/cpptcl_runtime.c)
+target_link_libraries(cpptcl_runtime ${TCL_LIBRARY})
+
```

Can we get a static library too?  Firehose is using all statics so it can just create a "fat" binary.
```suggestion
add_library(cpptcl_runtime_static STATIC ${cpptcl_SOURCE_DIR}/cpptcl_runtime.c)
target_link_libraries(cpp_runtime_static ${TCL_LIBRARY}
```

---

Change to a static library.